pkgs/development/python-modules/submitit/default.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  wheel,
+  cloudpickle,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "submitit";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "facebookincubator";
+    repo = "submitit";
+    rev = version;
+    hash = "sha256-N54qbj4aI5R4cbMxyQ84hkPUgZMSYGeWzDws50/X+bA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    cloudpickle
+    typing-extensions
+  ];
+
+  pythonImportsCheck = [
+    "submitit"
+  ];
+
+  meta = {
+    description = "Python 3.8+ toolbox for submitting jobs to Slurm";
+    homepage = "https://github.com/facebookincubator/submitit";
+    license = lib.licenses.mit;
```

```suggestion
    homepage = "https://github.com/facebookincubator/submitit";
    changelog = "https://github.com/facebookincubator/submitit/releases/tag/${version}";
    license = lib.licenses.mit;
```

Same for `torchbench`.